galaxy.yml

```
@@ -9,7 +9,7 @@ namespace: azure
 name: azcollection
 
 # The version of the collection. Must be compatible with semantic versioning
-version: 1.14.0
+version: 1.15.0
```

Please keep this version, we will only change this version when the version is released! Thanks!
```suggestion
version: 1.14.0
```